### PR TITLE
Feature/arpa 272

### PR DESCRIPTION
**Pull request by VILLAN3LL3**
Added properties and navigation tables for musician profiles

### CODE REVIEW

File: Orso.Arpa.Domain/Entities/SphereOfActivityConcert.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+using System;
+
+namespace Orso.Arpa.Domain.Entities
+{
+    public class SphereOfActivityConcert : BaseEntity
+    {
+        public SphereOfActivityConcert(Guid? id, MusicianProfile musicianProfile, Venue venue) : base(id)
```
Comment (euphi967) [inline Orso.Arpa.Domain/Entities/SphereOfActivityConcert.cs:7]: Wording: _Sphere of activity_ sounds spacy to me. I'd rather choose _area of activity_ of _activity area_

File: Orso.Arpa.Domain/Logic/SelectValueCategories/SelectValueCategorySeedData.cs [hunk 2]
```diff
@@ -23,140 +23,116 @@ public static IList<SelectValueCategory> SelectValueCategories
                     ProjectGenre,
                     ProjectType,
                     ProjectState,
+                    MusicianProfileQualification,
+                    MusicianProfileSalary,
+                    MusicianProfileInquery,
```
Comment (euphi967) [inline Orso.Arpa.Domain/Logic/SelectValueCategories/SelectValueCategorySeedData.cs:28]: what is ment by Inquery?

File: Tests/Orso.Arpa.Api.Tests/IntegrationTests/AppointmentsControllerTests.cs [hunk 3]
```diff
@@ -274,7 +274,7 @@ public async Task Should_Set_Dates()
             staffParticipation.Participation = null;
             expectedDto.Participations.Add(staffParticipation);
             AppointmentParticipationListItemDto adminParticipation = AppointmentDtoData.AdminParticipation;
-            adminParticipation.MusicianProfiles.Add(new MusicianProfileDto { IsProfessional = false, SectionName = "Soprano 2" });
+            adminParticipation.MusicianProfiles.Add(new MusicianProfileDto { SectionName = "Soprano 2" });
```
Comment (euphi967) [inline Tests/Orso.Arpa.Api.Tests/IntegrationTests/AppointmentsControllerTests.cs:277]: please check your test, does my change fully heal the test or is something missing now?

**Review by euphi967 — CHANGES_REQUESTED** [review]
UsersControllerTests.Should_Delete_User fails with an InternalServerError now

File: Orso.Arpa.Api/Controllers/SectionsController.cs [hunk 4]
```diff
@@ -26,9 +26,9 @@ public SectionsController(ISectionService sectionService)
         [Authorize(Policy = AuthorizationPolicies.HasRolePolicy)]
         [HttpGet]
         [ProducesResponseType(StatusCodes.Status200OK)]
-        public async Task<ActionResult<IEnumerable<SectionDto>>> Get()
+        public async Task<ActionResult<IEnumerable<SectionDto>>> Get([FromQuery]bool instrumentsOnly = false)
```
Comment (euphi967) [inline Orso.Arpa.Api/Controllers/SectionsController.cs:29]: what's the idea behind _instrumentsOnly_? When looking into the SectionSeedData, e.g. a Trumpet is flagged as an instrument, a Flugelhorn or a Cornet is not.

Comment (VILLAN3LL3) [reply Orso.Arpa.Api/Controllers/SectionsController.cs:29]: We distinguish between main instrument (selectable from all sections with the IsInstrument Flag set to true) and doubling instruments which can be all of the child sections of the main instrument.

File: Tests/Orso.Arpa.Api.Tests/IntegrationTests/AppointmentsControllerTests.cs (hunk 3 above)
Comment (VILLAN3LL3) [reply Tests/Orso.Arpa.Api.Tests/IntegrationTests/AppointmentsControllerTests.cs:277]: Tests have passed now

**Comment by VILLAN3LL3** [comment]
> 
> 
> UsersControllerTests.Should_Delete_User fails with an InternalServerError now

Fixed :)

**Review by euphi967 — APPROVED** [review]

**Pull request merged**